components/shared/components/Widget/components/panes/PaymentPane/Vipps/VippsPane.tsx

```diff
@@ -34,12 +31,12 @@ export const VippsPane: React.FC<{
         {recurring === RecurringDonation.RECURRING && (
```

Comment:
Perhaps we should refactor these into separate files?
Likely makes the logic easier to follow. Potentially change it into a switch statement like you did before.
It's only two cases, but I feel like it's easier to follow, especially when there is a lot of code in the conditional renders.

Reply:
That is, have recurring and single in separate files.